Modules/core/Controller/CoreusersController.php

```diff
@@ -125,14 +125,36 @@ public function editAction($id) {
         }
         $script = "";
         if ($form->check()) {
-            if (!$id && $modelUser->isLogin($this->request->getParameter('login'))) {
-                throw new PfmException(CoreTranslator::LoginAlreadyExists($lang), 403);
+            $canEditUser = true;
+            if (!$id) {
+                if ($modelUser->isLogin($this->request->getParameter('login'))) {
+                    $canEditUser = false;
+                    throw new PfmException(CoreTranslator::LoginAlreadyExists($lang), 403);
+                }
+                if($modelUser->isEmail($form->getParameter("email"))) {
+                    // if email alreday exists, warn user
+                    $canEditUser = false;
+                    $_SESSION["flash"] = CoreTranslator::EmailAlreadyExists($lang);
+                }
             } else {
+                $emailUsedByAnother = $modelUser->isEmailWithFilter($form->getParameter("email"), $user["email"]);
```

Comment:
why a filter ?
if param email == user['email'], that's fine (no change)
if different, should be used nowhere so isEmail should be enough

Reply:
ah, yes. It's simpler